packages/website/public/schema.yml

```
@@ -0,0 +1,341 @@
+openapi: 3.0.0
+info:
+  title: nftstorage.link API
+  version: '1.0'
+
+servers:
+  - url: https://api.nftstorage.link
+  - url: http://127.0.0.1:8787
+
+tags:
+  - name: nftstorage.link
+paths:
+  /perma-cache/{url}:
+    post:
+      tags:
+        - nftstorage.link
+      summary: Put a given URL response in nftstorage.link perma-cache.
```

I would avoid using the word "Put" for this HTTP POST route.

```suggestion
      summary: Add the content of the given URL to the perma-cache.
```